apps/condo/domains/organization/schema/Organization.test.js

```diff
@@ -30,20 +28,31 @@ describe('Organization', () => {
         expect(data).toEqual({ meta: { count: null } })
     })
 
-    test('user: allow to getAll', async () => {
+    test('user: can read only organizations, it employed in', async () => {
         const admin = await makeLoggedInAdminClient()
-        await createOrganization(admin)
-        const [, userAttrs] = await createTestUser(admin)
-        const client = await makeLoggedInClient(userAttrs)
+        await createTestOrganization(admin)
+        const client = await makeClientWithRegisteredOrganization()
+
         const objs = await Organization.getAll(client, {})
-        expect(objs.length).toBeGreaterThan(0)
+        expect(objs.length).toBe(1)
```

Comment:
You can also check that it's exactly the user organization like so:
`expect(objs[0].id).toEqual(client.organization.id)`

Reply:
Added in `aac0b8d3`